packages/configuration-validation/package.json

```diff
@@ -35,5 +35,8 @@
     "babel-cli": "^6.26.0",
     "eslint": "^4.7.1",
     "jest": "^23.6.0"
+  },
+  "jest": {
```

Comment:
Have to add this in all jest configs, due to a known issue - https://github.com/facebook/jest/issues/7064